XpressAI/xpipes
# Rename Xpipes to Xircuits

Welcome to Xpipes! Thank you for making a pull request. Please ensure that your pull request follows the template.

# Description

The repo name is changed from Xpipes to Xircuits. All related `xpipes` variables and strings have been updated to `xircuits`. I have also updated the `src/xai_components` folder to 'src/tray_library` to reflect the contents of the folder.

## Pull Request Type


## Type of Change


# Tests

1. Ensure Build is Successful

Simply run the usual build steps, if you're on windows git bash:
```
git clone https://github.com/XpressAI/xpipes &&
cd xpipes &&
git checkout fahreza/xpipes2xircuits &&
python -m venv venv &&
source venv/Scripts/activate &&
pip install -r requirements.txt &&
pip install -e . &&
jupyter labextension develop . --overwrite &&
jupyter server extension enable xircuits
```
Check the usual features, some from [here](https://docs.google.com/spreadsheets/d/158-vQ3sFknn0Kd4M5vNVCabWzlAglk-xh8uFOE1dhHQ/edit#gid=0). Aside from the type checking, everything should work. 

2. Ensure wheel is Successful

I've attached the wheel with this pull request. Give it a try. 
[dist.zip](https://github.com/XpressAI/xpipes/files/7931611/dist.zip)


## Tested on?


# Notes
